AEFeinstein/Super-2024-Swadge-FW
# Theme menu

### Description

Replaced `menuLogbookRenderer` with `menuManiaRenderer`. It's themed and functional, but not perfect yet.

Note that #219 still has TODOs, but I want to get this in `main` to set the vibe. I also may not be able to circle back on this for a while.

![image](https://github.com/AEFeinstein/Super-2024-Swadge-FW/assets/231180/5625c2f3-b60c-494c-a4a7-cdfbb7f4118a)

### Test Instructions

Run firmware and emulator, scroll through the settings menu to see all the arrows

### Ticket Links

#219

### Readiness Checklist
